contract/Chess.sol

```diff
@@ -151,10 +151,13 @@ contract Chess is TurnBasedGame {
             throw;
         if (msg.sender == game.nextPlayer)
             throw;
-        if (game.timeoutState == 0)
-            throw;
         if (now < game.timeoutStarted + 10 minutes)
             throw;
+        if (game.timeoutState == 0)
+            game.ended = true;
+            game.winner = msg.sender;
+            GameEnded(gameId, msg.sender);
+
```

Comment:
Good point with the one-line blocks and braces. But what the hag is this commit??